.rubocop.yml

```diff
@@ -154,3 +154,78 @@ Style/RegexpLiteral:
   EnforcedStyle: slashes
   Exclude:
     - 'spec/cucumber/project_initializer_spec.rb'
+
```

Comment:
Use NewCops and save yourself time here.

See: https://github.com/cucumber/cucumber-rails/blob/master/.rubocop.yml#L10